tutorial/markdown/python/python-langchain-pdf-chat-query/python-langchain-pdf-chat-query.md

```diff
@@ -0,0 +1,593 @@
+---
+# frontmatter
+path: "/tutorial-python-langchain-pdf-chat-query"
+# title and description do not need to be added to markdown, start with H2 (##)
+title: Build PDF Chat App With Couchbase Python SDK, LangChain and Vector Search
+short_title: Build PDF Chat App (Vector Search)
+description:
+  - Construct a PDF Chat App with LangChain, Couchbase Python SDK, Couchbase Vector Search (Query Service), and Streamlit.
+  - Learn to upload PDFs into Couchbase Vector Store with LangChain using Query-based Vector Search.
+  - Discover how to use RAG's for context-based Q&A's from PDFs with LLMs.
+content_type: tutorial
+filter: sdk
+technology:
+  - hyperscale vector index
+  - composite vector index
+  - kv
+tags:
+  - Streamlit
+  - LangChain
+  - OpenAI
+  - Artificial Intelligence
+sdk_language:
+  - python
+length: 45 Mins
+---
+
+## Introduction
+
+Welcome to this comprehensive guide on constructing an AI-enhanced Chat Application using Couchbase Vector Search. We will create a dynamic chat interface capable of delving into PDF documents to extract and provide summaries, key facts, and answers to your queries. By the end of this tutorial, you'll have a powerful tool at your disposal, transforming the way you interact with and utilize the information contained within PDFs.
```

Comment:
Can you add a paragraph hhighlighting that this tutorial is using the Query Based Vector Search. If you are looking for Vector Search using Search (fka FTS) service, refer to this tutorial instead.